TwoDo/app/src/main/java/com/example/twodo/TaskFragment.kt

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License.
+ */
+
+package com.example.twodo
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Button
+import android.widget.CheckBox
+import android.widget.EditText
+import android.widget.TextView
+import android.widget.Toast
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.activityViewModels
+import androidx.lifecycle.Observer
+import androidx.slidingpanelayout.widget.SlidingPaneLayout
+
+class TaskFragment : Fragment(), View.OnClickListener {
+
+    private val taskViewModel: TaskViewModel by activityViewModels()
+
+    private lateinit var title: TextView
+    private lateinit var taskName: EditText
+    private lateinit var taskNotes: EditText
+    private lateinit var taskComplete: CheckBox
+
+    private lateinit var task: Task
+
+    lateinit var slidingPane: SlidingPaneLayout
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+    }
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+
+        // Inflate the layout for this fragment
+        val view = inflater.inflate(R.layout.fragment_task, container, false)
+
+        // task fields
+        title = view.findViewById<TextView>(R.id.create_edit_title)
```

Comment:
You can use [ViewBindings](https://developer.android.com/topic/libraries/view-binding) too here instead of `findViewById`